### ISSUE TITLE
Gamedev section for the rust-lang.org website.

### ISSUE THREAD

**Issue by AlexEne** [rust-gamedev/wg#13]
I am opening this issue here, but the output should be a PR to the rust-lang.org repository here: https://github.com/rust-lang/www.rust-lang.org

Good example:  
https://www.rust-lang.org/what/embedded

I propose we have a few quotes from Chucklefish / Embark / others + the description at the top with why Rust. (basically everything without get started for now).

Once the book and guide proposed are in a good shape with stable links we can also add them in a follow-up change.

**Comment by Lokathor** [comment]
I agree, but as a minor point, Chucklefish no longer does much _game dev_ in Rust, because they had basically one person pushing Rust at the company and they left the company.

As I understand it, they have non-game tech written in Rust, but no games released or planned to be released in Rust.

**Comment by AlexEne** [comment]
I didn't know that, but Ready at Dawn, Embark, are also good candidates.

**Comment by erlend-sh** [comment]
I still think the Chucklefish story is worth showcasing. It just needs a proper disclaimer:

> NOTE: While Chucklefish has since opted to drop their Rust experiment to focus on their existing tooling, their use case for Rust in development remains valid.
